src/shogun/regression/LeastAngleRegression.h

```diff
@@ -184,20 +188,49 @@ class CLeastAngleRegression: public CLinearMachine
 	virtual const char* get_name() const { return "LeastAngleRegression"; }
 
 protected:
-	virtual bool train_machine(CFeatures* data=NULL);
+	/**
+	* An interface method to - this is called by the superclass's (CLinearMachine)
+	* train() method.  This method checks to see if data is a dense feature vector,
+	* and that it's elements are of type float64_t or float32_t.  It then calls 
```

Comment:
try to avoid explicitly mentioning the supported types since they easily get outdated. I.e. here oyu talk about 32 and 64, but not about the recently added floatmat. So better completely avoid such things